lib/msal-common/src/app/module/AuthModule.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * @hidden
+ * @ignore
+ * Data type to hold information about state returned from the server
+ */
+export type ResponseStateInfo = {
```

Comment:
should this be in a different file?

Reply:
This could potentially go in the respone handler file, I will move it when I release the PR for the response handling.